docker-compose.yaml

```
@@ -45,6 +45,9 @@ services:
       - DATABASE_USER=${DATABASE_USER}
       - DATABASE_PASSWORD=${DATABASE_PASSWORD}
       - DATABASE_URL=${DATABASE_URL}
+      - CONFIG_DATABASE_USER=${CONFIG_DATABASE_USER}
+      - CONFIG_DATABASE_PASSWORD=${CONFIG_DATABASE_PASSWORD}
+      - CONFIG_DATABASE_URL=${CONFIG_DATABASE_URL}
```

This will throw warnings when running `docker-compose up` if the variables are not defined in the `.env` file, maybe it's worth setting default values as `""` in `.env` instead?

or explicitly express this in the `docker-compose`:
```suggestion
      - CONFIG_DATABASE_USER=${CONFIG_DATABASE_USER:-$DATABASE_USER}
      - CONFIG_DATABASE_PASSWORD=${CONFIG_DATABASE_PASSWORD:-$DATABASE_PASSWORD}
      - CONFIG_DATABASE_URL=${CONFIG_DATABASE_URL:-$DATABASE_URL}
```

---

This somehow set `CONFIG_DATABASE_USER` to literally `$DATABASE_USER`, instead of the value of that variable.